pkg/service/ecloud/request.go

```diff
@@ -245,7 +245,7 @@ type CreateInstanceRequest struct {
 	BackupEnabled      bool                   `json:"backup_enabled"`
 	NetworkID          string                 `json:"network_id,omitempty"`
 	FloatingIPID       string                 `json:"floating_ip_id,omitempty"`
-	RequiresFloatingIP bool                   `json:"requires_floating_ip"`
+	RequiresFloatingIP *bool                  `json:"requires_floating_ip,omitempty"`
```

Comment:
We should be able to add `omitempty` without changing bool to pointer, as we would only ever want to send this over if true

Reply:
ah yeah you're right :)